facebook/delphiFacebook/R/contingency_variables.R

```diff
@@ -979,13 +1001,11 @@ create_derivative_columns <- function(df) {
     df$b_hesitant_barrier_pregnant <- as.numeric(is_selected(hesitancy_reasons, "14"))
     df$b_hesitant_barrier_religious <- as.numeric(is_selected(hesitancy_reasons, "15"))
     
-    if (wave >= 11) {
-      input_data$b_hesitant_barrier_allergic <- NA
-      input_data$b_hesitant_barrier_not_recommended <- NA
-      input_data$b_hesitant_barrier_distrust_vaccines <- NA
-      input_data$b_hesitant_barrier_health_condition <- NA
-      input_data$b_hesitant_barrier_pregnant <- NA
-    }
+    df$b_hesitant_barrier_allergic[df$wave >= 11] <- NA
```

Comment:
I feel like there should be a comment here stating that these must explicitly be set these to missing because `is_selected` will always return false if an option isn't selected but there is an answer to the question, even if the option is no longer available. It took me a little bit to understand.